charts/scality-mountpoint-s3-csi-driver/templates/serviceaccount-csi-controller.yaml

```diff
@@ -13,34 +13,6 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 ---
-kind: Role
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: s3-csi-driver-controller-role
-  namespace: {{ .Values.mountpointPod.namespace }}
-  labels:
-    {{- include "scality-mountpoint-s3-csi-driver.labels" . | nindent 4 }}
-rules:
-  - apiGroups: [""]
-    resources: ["pods"]
-    verbs: ["get", "create", "watch", "delete", "list"]
----
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: s3-csi-driver-controller-role-binding
-  namespace: {{ .Values.mountpointPod.namespace }}
-  labels:
-    {{- include "scality-mountpoint-s3-csi-driver.labels" . | nindent 4 }}
-subjects:
-  - kind: ServiceAccount
-    name: {{ .Values.controller.serviceAccount.name }}
-    namespace: {{ .Release.Namespace }}
-roleRef:
-  kind: Role
-  name: s3-csi-driver-controller-role
-  apiGroup: rbac.authorization.k8s.io
----
```

Comment:
We do not need this for now, this will be needed when we support se linux in v2 architecture using pod mounter